src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label>
+            Surname:
+            <input 
+              type="text" 
+              name="surname"
+              autocomplete="off"
+              pattern="\w+"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">          
+          <label>
+            Name:
+            <input 
+              type="text" 
+              name="name"
+              autocomplete="off"
+              pattern="\w+"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            How old are You?
+            <input 
+              type="number" 
+              value="12" 
+              name="age"
+              min="1"
+              max="100"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Full date of birth:
+            <input 
+              type="date" 
+              name="date_of_birth"
```

Comment:
[checklist p.4 ](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)Use camelCase for values of name attribute

Reply:
not fixed

Reply:
done